app/models/benefit_model.rb

```diff
@@ -22,7 +31,13 @@ def benefit_options
     FREQUENCY_OPTIONS.map { [_1, I18n.t("estimate_flow.benefits.frequencies.#{_1}")] }
   end
 
+  def cfe_benefit_list
+    CfeConnection.connection.state_benefit_types
+  end
+
   def benefit_list
-    @benefit_list ||= CfeConnection.connection.state_benefit_types
+    full_list = cfe_benefit_list.pluck("label", "name", "exclude_from_gross_income")
+    display_list = full_list.reject { |item| item[2] == true || item[0].in?(PASSPORTED_BENEFITS) }
```

Comment:
Another aesthetic thing: you don't really need the `pluck`. You could do 

```ruby
display_list = cfe_benefit_list.reject { _1["exclude_from_gross_income"] || _1["label"].in?(PASSPORTED_BENEFITS }
display_list.map { _1["name"] }
```

And that would mean whenever you use an attribute you can refer to it by its name rather than a number.